Rakefile

```diff
@@ -51,3 +51,22 @@ task update_books: :environment do
     end while done == false && i < 5
   end
 end
+
+LEVELS_CONVERT = YAML.load(File.read(File.expand_path('../db/levelsConvert.yml', __FILE__)))
+
+task add_level_tags: :environment do
+  puts "Adding human-readable level tags"
+
+  Book.all.each do |book|
```

Comment:
It's a little wasteful for this to be done for all books every time. Maybe have a field on book that checks if the levels have been added once and only do it for books that don't have that field.